### PR TITLE
OKTA-567644 API updates for X509_BASED_AUTHENTICATOR

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add properties.additionalAmr property to the IdP API for Smart Card authenticator
- **Is this PR related to a Monolith release?** Yes, doc' with RN in 2023.02.0 (orig in 2023.01.1)

### Preview:
https://63e30e0ff6482c0c81ad33f1--reverent-murdock-829d24.netlify.app/docs/reference/api/idps/#identity-provider-properties

### Resolves:

* [OKTA-567644](https://oktainc.atlassian.net/browse/OKTA-567644)

### CODE REVIEW

**Comment by vanngo-okta** [comment]
@valberbenetz-okta:
Question about the `properties` property - is it EA? Or is it already available in GA for other IdP types?

Can you please provide me with an updated request/response example for “Add Smart Card IdP”? The old example is located here: https://developer.okta.com/docs/reference/api/idps/#add-smart-card-identity-provider

Also, we don’t have a Smart Card Authenticator object example. Can you please provide an example to be placed in this section: https://developer.okta.com/docs/reference/api/authenticators-admin/#example-email-authenticator

Thank you!

**Review by shantanusardal-okta — APPROVED** [review]
looks good from Federations side

File: packages/@okta/vuepress-site/docs/reference/api/idps/index.md [hunk 1]
```diff
@@ -4825,6 +4852,14 @@ Okta supports the following enterprise and social Identity Provider types:
 | `YAHOO`      | [Yahoo](https://login.yahoo.com/)&nbsp;as the Identity Provider | [OpenID Connect](#openid-connect-protocol) | `openid`, `profile`, `email` |
 | `YAHOOJP`      | [Yahoo Japan](https://login.yahoo.co.jp/config/login)&nbsp;as the Identity Provider | [OpenID Connect](#openid-connect-protocol) | `openid`, `profile`, `email` |
 
+### Identity Provider Properties
+
+The properties in the Identity Provider Properties object varies depending on the IdP type.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/idps/index.md:4857]:
```suggestion
The properties in the Identity Provider Properties object vary depending on the IdP type.
```

**Review by susanharper-okta — APPROVED** [review]
one issue. approving to not hold you up.

**Pull request merged**